### DISCUSSION TITLE
Changes for arabic taiwanese

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#353] [Archived]
<USER>>>> Pak<br/> <DATE> [February 10, 2021, 12:17pm]<br/> <br/> Hello, *slash* <br/> I try use: DDC-TTS_Universal-Fullband-MelGAN-LJSpeech_EN.ipynb *slash* <br/> But want train labeled data in Arabic. What changes need make? Must<br/> modify cleaners? *slash* <br/> Thanks you<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/changes-for-arabic-taiwanese]<br/>

**Comment by JRMeyer** [comment]
## [Archived] Changes for arabic taiwanese<br/> <br/> <USER>>>> Pak<br/> <DATE> [February 10, 2021, 12:17pm]<br/> <br/> Hello, *slash* <br/> I try use: DDC-TTS_Universal-Fullband-MelGAN-LJSpeech_EN.ipynb *slash* <br/> But want train labeled data in Arabic. What changes need make? Must<br/> modify cleaners? *slash* <br/> Thanks you<br/> <br/> ### This is an archived TTS discussion thread from discourse.mozilla.org/t/changes-for-arabic-taiwanese<br/>

**Reply by cod3r0k** [reply]
Hi do you have any update for Arabic language? We must remove diacritics and others or not? @JRMeyer
